src/drivers/Cst816s.cpp

```diff
@@ -56,28 +56,33 @@ bool Cst816S::Init() {
   static constexpr uint8_t irqCtl = 0b01110000;
   twiMaster.Write(twiAddress, 0xFA, &irqCtl, 1);
 
+  // Disable auto-reset after 5s of no gesture
+  // The reset kills the current touch point, so paint/pong etc breaks
+  twiMaster.Write(twiAddress, 0xFB, 0, 1);
+
   return true;
 }
 
 Cst816S::TouchInfos Cst816S::GetTouchInfo() {
   Cst816S::TouchInfos info;
-  uint8_t touchData[7];
+  uint8_t touchData[6];
```

Comment:
while editing this, we could port to `std::array<unit8_t, 6>` :thinking:

Reply:
Good idea